lib/config/react.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  'extends': ['./base', 'tslint-react'],
```

Comment:
This seems weird.  The react config is extending itself?  Any reason to not just extend `./base`?